avconv.js

```diff
@@ -160,7 +221,7 @@ module.exports = function avconv(params) {
 
     avconv.on(eventType, function(exitCode, signal) {
         stream.end();
-        stream.emit('exit', exitCode, signal);
+        stream.emit('exit', exitCode, signal, parseMetaData(output));
```

Comment:
well, you are right, but i have a time sensitive app here using this package.

tell me how long it takes to parse meta data?